contribute/backend/style-guide.md

```diff
@@ -69,6 +69,7 @@ Use [`t.Cleanup`](https://golang.org/pkg/testing/#T.Cleanup) to clean up resourc
 Optionally, we use [`mock.Mock`](https://github.com/stretchr/testify#mock-package) package to write mocks. This is
 useful when you expect different behaviours of the same function.
 
+
```

Comment:
- Use semantic line breaks for better line based diffing in GitHub UI in the future.
- Use American English.
- Fix `prettier` formatting lint issue.

```suggestion
Optionally, we use [`mock.Mock`](https://github.com/stretchr/testify#mock-package) package to write mocks. 
This is useful when you expect different behaviors of the same function.

```